cfme/tests/containers/test_containers_default_services.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from utils import testgen
+from utils.appliance.implementations.ui import navigate_to
+from utils.version import current_version
+from cfme.containers.service import Service, list_tbl
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+
+# CMP-9448
```

Comment:
I think it would be good to put this near the ``def`` so it would be more clear does it belong to the whole file or the test method